public/pages/Findings/containers/Findings/Findings.tsx

```diff
@@ -206,17 +327,50 @@ class Findings extends Component<FindingsProps, FindingsState> {
     this.setState({ loading: false });
   };
 
+  getThreatIntelFindings = async () => {
+    try {
+      this.abortGetFindings();
+      this.setStateForTab(
+        {
+          tabId: FindingTabId.ThreatIntel,
+          field: 'findings',
+          value: [],
+        },
+        {
+          loading: true,
+        }
+      );
+      const duration = this.props.dateTimeFilter
+        ? getDuration(this.props.dateTimeFilter)
+        : undefined;
+      const abortController = new AbortController();
+      this.abortGetFindingsControllers.push(abortController);
+      await DataStore.threatIntel.getThreatIntelFindings(
+        abortController.signal,
+        duration,
+        this.onStreamingThreatIntelFindings
+      );
+    } catch (e: any) {}
```

Comment:
```
catch (error) {
    console.error('Error fetching threat intel findings:', error);
    // notificstionToast? (e.g., show error message to user)
  } finally {
    this.setState({ loading: false }); // Always set loading to false after async operation
  }

```

Reply:
Need to add error handling overall, will do in a quick follow up PR